Code/Editor/EditorViewportWidget.cpp

```diff
@@ -618,6 +621,10 @@ void EditorViewportWidget::OnEditorNotifyEvent(EEditorNotifyEvent event)
 
 void EditorViewportWidget::OnBeginPrepareRender()
 {
+    AZ_PROFILE_FUNCTION(Editor);
```

Comment:
Why `Editor` for the profile function? doesn't seem to represent the begin prepare of the render.